collector/diskstats_linux.go

```diff
@@ -65,6 +66,10 @@ const (
 	udevSCSIIdentSerial         = "SCSI_IDENT_SERIAL"
 )
 
+var (
+	disableQueueStats = kingpin.Flag("collector.diskstats.disable-queue-stats", "disable queue stats").Default("false").Bool()
```

Comment:
We usually prefer "positive" flags. Then the disable would be `--no-collector.diskstats.enable-queue-stats`

This makes the explicit version of the disable less awkward looking (`--no-collector.diskstats.disable-queue-stats`).

```suggestion
	enableQueueStats = kingpin.Flag("collector.diskstats.enable-queue-stats", "enable queue stats").Default("true").Bool()
```